packages/xdl/src/Analytics.ts

```diff
@@ -9,59 +9,59 @@ const PLATFORM_TO_ANALYTICS_PLATFORM: { [platform: string]: string } = {
   linux: 'Linux',
 };
 
-let _userId: string | undefined;
-let _userTraits: any;
-
 export class AnalyticsClient {
-  private segmentNodeInstance: Segment | undefined;
-  private version: string | undefined;
-  private userIdentifyCalled: boolean = false;
+  private _userId: string | undefined;
+  private _userTraits: any;
+  private _segmentNodeInstance: Segment | undefined;
+  private _version: string | undefined;
```

Comment:
in typescript with private/public specifiers using _ is unnecessary (unless you need getter with the same name, so maybe remove _ where possible. Some code in xdl is still written this way but it's mostly because it was originally written in js

Reply:
the idea was to keep a consistent naming convention within the class.  Since the getters / private field names were colliding i decided on `_` for private and no prefix for public.  As you suggested I removed the prefix where there was not a collision.